File: trunk/server/src/main/java/org/ngame/protocol/OnlineResponse.java
/**
 * 用来表达协议的返回
 */
package org.ngame.protocol;

import java.util.List;

import org.ngame.protocol.annotation.Proto;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

import lombok.Data;

/**
 *
 * @author beykery
 */
@Data
@Proto(value=-9999,description = "这是一条服务器的转发")
public class OnlineResponse
{
	  @Protobuf(description = "online pids",fieldType=FieldType.STRING)
	  private List<String> pids;
}

File: trunk/server/net/Protocols/OnlineResponse.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// 这是一条服务器的转发 协议:-9999
    /// </summary>
	[Proto(value=-9999,description="这是一条服务器的转发")]
	[ProtoContract]
	public class OnlineResponse
	{
        /// <summary>
        ///  online pids
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public List<string> pids;

	}
}